R/findC.R

```diff
@@ -27,7 +27,8 @@ findC <- function(timepoints, max.lag = NULL, pi = 0.95, iter = 10){
     max.lag <- floor(length(timepoints) / 4)
   }
   #checking through all the conditions
-  stopifnot(all(is.numeric(timepoints)), max.lag <= length(timepoints) / 4, max.lag %% 1 == 0)
+  stopifnot(all(is.numeric(timepoints)), max.lag <= length(timepoints) / 4, max.lag %% 1 == 0,
+            (pi > 0.5 & pi < 1), iter > 1, iter %% 1 == 0)
```

Comment:
`corr_bestlag.R` should also require `iter > 1`.

Reply:
Again, this would be redundant. I can add it if you think its necessary.

Reply:
I made the changes anyway. We had a lot of redundancy.